_posts/2016/2016-07-26-xconf-2016.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "XConf 2016 Hamburg"
+date: "2016-07-26 00:00:00"
+image: blog-header/lego.jpeg
+categories: events
+authors: ["Oleksandr V", "Renato"]
+---
+
+Not so long time ago, in one of so few sunny days in Hamburg I have had an opportunity to visit
+[XConf](https://info.thoughtworks.com/Xconf-hamburg-2016.html). Here in ePages, developers have a great support by the company if they are interested in self-education.
+
+This yearly event took place on 12th of July in a cozy place [Betahaus](http://hamburg.betahaus.de/) in format of one day conference covering "broad range of technology topics".
+
+Thanks to the great job made by [ThoughtWorks](https://www.thoughtworks.com/) I have enjoyed time participating it.
+
+It was a surprise for me to meet another developer from my company deciding to visit this conference as well.
+
+{% image blog/blog-twitter-xconf-2.png %}
+
+## Keynote
+
+[James Lewis](https://twitter.com/boicy) presented the keynote talking about Lean Software Engineering, [LEGO](http://lego.com) and [Star Wars](http://www.starwars.com/).
+Sometimes when people present processes they make theirs presentation too abstract and might loose the audience on the way.
+What was interesting in this presentation, that he was able to illustrate in a form of a story, processes problems and discussed ways of solving them.
```

Comment:
But James presented the keynote in form of a story, processed problems, and discussed ways of solving them.